Azure/azure-cli
# [RDBMS] Add Azure active directory administrator commands for PostgreSQL and MySQL

**Description of PR (Mandatory)**  
introduce ossrdbms azure active directory administrator commands for Postgres and Mysql

**Testing Guide**  
Tested with the below set of commands
Set the Adming

az postgres server ad-admin create --name <server name>  -g <resoure_group>  --display-name username@domain.com  --object-id <sid>

az mysql server ad-admin create --name <server name>  -g <resoure_group>  --display-name username@domain.com  --object-id <sid>

Get the admin

az postgres server ad-admin show --name <server name>  -g <resoure_group> 
az mysql server ad-admin show --name <server name>  -g <resoure_group> 

List the admin

az postgres server ad-admin list --name <server name>  -g <resoure_group> 
az mysql server ad-admin list --name <server name>  -g <resoure_group> 

Drop the admin

az postgres server ad-admin delete --name <server name>  -g <resoure_group> 
az mysql server ad-admin delete --name <server name>  -g <resoure_group> 


-

This checklist is used to make sure that common guidelines for a pull request are followed.
